packages/admin-ui-docs/content/grid/code.mdx

```diff
@@ -0,0 +1,86 @@
+import { Section } from '@site/src/components/Section'
+
+<Section>
+
+## Usage
+
+```jsx
+import { Grid, GridItem } from '@vtex/admin-ui'
+
+function Example() {
+  return (
+    <Grid
+      templateAreas={[
+        'item-1 item-2 item-3',
+        'item-4 item-5 item-6',
+        'item-7 item-8 item-9',
+      ]}
+      templateColumns="repeat(3, 1fr)"
+      templateRows="repeat(3, 1fr)"
+      gap="6"
+    >
+      <GridItem area="item-1" />
+      <GridItem area="item-2" />
+      <GridItem area="item-3" />
+      <GridItem area="item-4" />
+      <GridItem area="item-5" />
+      <GridItem area="item-6" />
+      <GridItem area="item-7" />
+      <GridItem area="item-8" />
+      <GridItem area="item-9" />
+    </Grid>
+  )
+}
+```
+
+</Section>
+
+<Section>
+
+## Composition
+
+| Name       | Description                                                                   | Props           |
+| ---------- | ----------------------------------------------------------------------------- | --------------- |
+| `Grid`     | The main wrapper with `display: grid` and helpful styles shorthand            | `GridProps`     |
+| `GridItem` | Used as a child of `Grid` to control the `gridArea` position whithin the grid | `GridItemProps` |
+
+</Section>
+
+<Section>
+
+## Props
+
+</Section>
+
+<Section>
+  
```

Comment:
I think there is a blank space here and it is avoid the **Grid** heading be rendered properly.